c/shared/source/uforead/uforead.c

```diff
@@ -1912,6 +1801,241 @@ static int parseFontInfo(ufoCtx h) {
     return parsingSuccess;
 }
 
+static bool parseFontInfoFDArray(ufoCtx h, xmlNodePtr cur) {
+    h->top.FDArray.array = memNew(h, FDArrayInitSize *sizeof(abfFontDict));
+    if (h->top.version.ptr != NULL)
+        h->top.cid.CIDFontVersion = atoi(h->top.version.ptr) % 10 + (float) atoi(&h->top.version.ptr[2])/1000;
```

Comment:
Moving forward with this code base we might want to switch to `strtol()` and check for errors with all of the `strtoX()` functions. (If you pass a char ** as the second argument they will set that to the pointer passed in if it couldn't parse a number.)

However, we don't do that now so I don't see a reason to hold up this change for that kind of improvement.